python/ray/autoscaler/commands.py

```diff
@@ -180,18 +188,76 @@ def get_or_create_head_node(config, no_restart, yes):
                                        provider.external_ip(head_node)))
 
 
-def get_head_node_ip(config_file):
+def attach_cluster(config_file, start, override_cluster_name):
+    """Attaches to a screen for the specified cluster."""
+
+    exec_cluster(config_file, "screen -L -xRR", False, False, start,
+                 override_cluster_name)
+
+
+def exec_cluster(config_file, cmd, screen, stop, start, override_cluster_name):
```

Comment:
The arguments for the functions in this file should be documented in the docstrings.

Reply:
Done